doc/releases/migration-guide-3.7.rst

```diff
@@ -540,6 +551,9 @@ Networking
   to receive all the network packets. See details in
   https://www.man7.org/linux/man-pages/man7/packet.7.html documentation. (:github:`73338`)
 
+* TCP now uses SHA256 instead of MD5 for ISN generation. The crypto support for
+  for this hash computations was also changed from Mbed TLS to PSA APIs. (:github:`71827`)
```

Comment:
```suggestion
* TCP now uses SHA256 instead of MD5 for ISN generation. The crypto support for
  this hash computation was also changed from Mbed TLS to PSA APIs. (:github:`71827`)
```